src/shared/sam/debugger/awsSamDebugConfiguration.ts

```diff
@@ -85,7 +87,16 @@ export function createTemplateAwsSamDebugConfig(
     return response
 }
 
-export function createCodeAwsSamDebugConfig(lambdaHandler: string, projectRoot: string): AwsSamDebuggerConfiguration {
+export function createCodeAwsSamDebugConfig(
+    lambdaHandler: string,
+    projectRoot: string,
+    runtimeFamily?: RuntimeFamily
+): AwsSamDebuggerConfiguration {
+    const runtime = runtimeFamily ? getDefaultRuntime(runtimeFamily) : undefined
+    if (!runtime) {
+        throw new Error('Invalid or missing runtime family')
```

Comment:
should the `runtimeFamily?` parameter not be optional?

Reply:
It's optional because it's optional on the level up (since template types don't care). Just depends on where you'd rather throw the error.